Doc/reference/lexical_analysis.rst

```diff
@@ -594,11 +594,8 @@ escape sequences only recognized in string literals fall into the category of
 unrecognized escapes for bytes literals.
 
    .. versionchanged:: 3.6
-      Unrecognized escape sequences produce a :exc:`DeprecationWarning`.
-
-   .. versionchanged:: 3.8
-      Unrecognized escape sequences produce a :exc:`SyntaxWarning`.  In
-      some future version of Python they will be a :exc:`SyntaxError`.
+      Unrecognized escape sequences produce a DeprecationWarning.  In
+      Python 3.9 they will be a SyntaxError.
```

Comment:
I don't think we should promise to raise `SyntaxError` in 3.9. This issue will no doubt lead to discussions again in the 3.9, so I wouldn't mention 3.9 and just drop that last sentence.

Reply:
This PR has been created in the assumption that SyntaxError will be kept in 3.9. If we decide other, it should be rebased against the master branch.

Reply:
Are you confusing `SyntaxWarning` with `SyntaxError`?

Reply:
Good catch!

Reply:
I modified the wording to not say "3.9" but instead say "future Python version" as we clearly don't have consensus on when and how this will move beyond DeprecationWarning so it is best not to promise a timeline.